.github/workflows/update-pkgdown.yml

```diff
@@ -18,6 +18,24 @@ on:
 
 name: update-pkgdown
 
+# Give the fewest permissions possible. content is necessary.
+# https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/controlling-permissions-for-github_token
+permissions:
+  contents: write
+  pull-requests: none

```

Comment:
Non-blocking suggestion/question: If you specify access for contents, do you need to explicitly set none for other permissions, or are unspecified permissions automatically set to none?

Reply:
Yes, I think you have to specifically set them to none for others- I think contents isn't everything, I think it is just the code part of the repository.

Reply:
actually - I think you are right - the docs say " If you specify the access for any of these permissions, all of those that are not specified are set to none." Thanks for pointing this out! Still, I think it is more clear to leave them all for now, if that is ok!

By the way, if you don't specify any of this, these are the defaults: https://docs.github.com/en/actions/security-for-github-actions/security-guides/automatic-token-authentication#permissions-for-the-github_token

Reply:
On second thought, I will simplify the permissions! I also did a test to confirm you don't need all the "none" lines. Thank you so much for the suggestion!